source/llvm/LLVMExecutableModel.cpp

```diff
@@ -138,6 +138,10 @@ int LLVMExecutableModel::setValues(bool (*funcPtr)(LLVMModelData*, int, double),
     {
         int j = indx ? indx[i] : i;
         bool result =  funcPtr(modelData, j, values[i]);
+        {
+          string id = (this->*getNameFuncPtr)(j);
+          Log(Logger::LOG_ERROR) << "LLVMExecutableModel::setValues for " << id << ", has initial assignment rule: " << symbols->hasInitialAssignmentRule(id);
```

Comment:
Why use LOG_ERROR rather than LOG_INFORMATION?

Reply:
I believe this is outdated and is no longer in `llvm-6`. Check out that [same line](https://github.com/sys-bio/roadrunner/blob/llvm-6/source/llvm/LLVMExecutableModel.cpp#L141) from the current `llvm-6` branch and the log statement is not there.